sources/packages/web/src/store/modules/student/actions.ts

```diff
@@ -17,4 +17,11 @@ export const actions: ActionTree<StudentState, RootState> = {
   ) {
     context.commit("setHasStudentAccount", hasStudentAccount);
   },
+
+  async updateStudentRestrictions(
+    context: ActionContext<StudentState, RootState>,
+  ) {
+    const restrictions = await StudentService.shared.getStudentRestriction();
+    context.commit("updateStudentRestrictions", restrictions);
```

Comment:
I noticed that the interface `StudentRestriction` was created but we are actually saving to the state the DTO, it is the expected behavior?